packages/theme/src/client/module/blog/components/TagList.ts

```diff
@@ -1,12 +1,12 @@
 import { usePageFrontmatter } from "@vuepress/client";
 import { computed, defineComponent, h } from "vue";
-import { RouterLink } from "vue-router";
 
 import { useTagMap } from "@theme-hope/module/blog/composables";
 
+
 import type { VNode } from "vue";
 import type { BlogPluginCategoryFrontmatter } from "vuepress-plugin-blog2";
-
+import Tag from "./Tag"
```

Comment:
You can directly import it using `import Tag from '@mr-hope/vuepress-plugin-components/lib/client/Tag'`